sdk/src/wallet/operations/syncing/mod.rs

```diff
@@ -18,13 +18,18 @@ use crate::{
     },
     wallet::{
         constants::MIN_SYNC_INTERVAL,
-        types::{address::AddressWithUnspentOutputs, Balance, OutputData},
+        types::{
+            address::{AddressWithUnspentOutputIds, AddressWithUnspentOutputs, SpentOutputId},
+            Balance, OutputWithExtendedMetadata,
+        },
         Wallet, WalletError,
     },
 };
 
 impl<S: 'static + SecretManage> Wallet<S> {
-    /// Set the fallback SyncOptions for account syncing.
+    // TODO: can we rename this to `set_fallback_sync_options`? It's no biggie, but if I read `default` in Rust
+    // if feels like it would use `SyncOptions::default()`, but that's not what we mean here, right?
```

Comment:
Yes it's not the Rust default, but it's also exposed to the bindings and might be weird there?

Reply:
not sure, but I am fine with not doing the change

Reply:
With the definition from here https://stackoverflow.com/a/76104060 it sounds like default would be the correct one :trollface:

Reply:
I agree with the reasoning in that answer, but it's not considering the Rust aspect. `fn default_sync_options(&self)` (method below) is confusing, but ... we could explicitly mention in the doc comment, that it doesn't return `SyncOptions::default()` necessarily.

Reply:
removed